src/main/kotlin/org/rust/ide/inspections/RsSimplifyBooleanExpressionInspection.kt

```diff
@@ -20,7 +20,7 @@ class RsSimplifyBooleanExpressionInspection : RsLocalInspectionTool() {
         override fun visitExpr(o: RsExpr) {
             if (o.isPure() != true)
                 return
-            val (expr, result) = o.simplifyBooleanExpression()
+            val (ignored, result) = o.simplifyBooleanExpression(true)
```

Comment:
At least my compiler haven't. Kotling plugin suggest to rename to `_`, but this does not compile for me.